plugins/Morpheus/javascripts/piwikHelper.js

```diff
@@ -122,15 +122,19 @@ var piwikHelper = {
             return;
         }
 
-        angular.element(document).injector().invoke(function($compile) {
-            var scope = angular.element($element).scope();
+        angular.element(document).injector().invoke(function($compile, $rootScope) {
```

Comment:
Options.scope was not working properly and not used. Fixed it and improved the api

Reply:
Yes that is what it means, I'll update the docs and clarify.